x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -445,6 +445,38 @@ class AgentPolicyService {
     return res;
   }
 
+  public async bumpAllAgentPoliciesForOutput(
```

Comment:
Let's go ahead and add a unit test for this logic to verify the correct policies are updated.